### PR TITLE
Object manipulation 3

### PR DESCRIPTION
**Pull request by 16hsuisaac**
image

### CODE REVIEW

File: object-manipulation-3/main.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+if (playerArray[2].score > playerArray[3].score) {
+  var winner2 = playerArray[2];
+} else {
+  winner2 = playerArray[3];
+}
+if (winner1.score > winner2.score) {
+  var finalWinner = winner1;
+} else {
+  finalWinner = winner2;
+}
```
Comment (codingcodymiller) [inline object-manipulation-3/main.js:50]: What if you had more than 4 players? You should find a way to do this that would work for any number of players

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start on this!! One note to get you started:

File: object-manipulation-3/main.js [hunk 2]
```diff
@@ -33,20 +33,14 @@ for (var o = 0; o < playerArray.length; o++) {
...
-}
-if (winner1.score > winner2.score) {
-  var finalWinner = winner1;
-} else {
-  finalWinner = winner2;
+var finalWinnerScore = 0;
+var finalWinner = null;
+
+for (var w = 0; w < playerArray.length; w++) {
+  if (playerArray[w].score > finalWinner) {
```
Comment (codingcodymiller) [inline object-manipulation-3/main.js:40]: Shouldn't this be compared to `finalWinnerScore`? `finalWinner` holds an object

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: object-manipulation-3/main.js (hunk 2 above)
Comment (16hsuisaac) [reply object-manipulation-3/main.js:40]: Ah, you're right

**Comment by 16hsuisaac** [comment]
image

**Review by codingcodymiller — APPROVED** [review]
Nicely done, dude!!!

**Pull request merged**